package.json

```diff
@@ -1,19 +1,23 @@
 {
-  "name": "branch-name",
+  "name": "branch-name-checker",
   "private": true,
   "main": "index.js",
   "scripts": {
     "start": "node ./index.js",
+    "build": "tsc",
```

Comment:
**suggestion (if-minor):** maybe instead of tsc we can make use of SWC or we can change it in a future iteration, SWC is another compiler for typescript that is so much faster than TSC: https://swc.rs/docs/getting-started

To use it we can:

1. Install swc: `npm i --save-dev @swc/cli @swc/core`
2. Use it in the script: `swc ./src -d dist --strip-leading-paths`

Reply:
I had thought of adding SWC or ESBuild in a second iteration.

Reply:
SWC in a second iteration sounds great, thanks!